File: server/src/Connection/Scheme/Position.cs
using System.Text.Json.Serialization;

namespace Thuai.Server.Connection;
public record Position
{
    [JsonPropertyName("x")]
    public double X { get; init; }

    [JsonPropertyName("y")]
    public double Y { get; init; }

    [JsonPropertyName("angle")]
    public double Angle { get; init; }
}

File: sdk/python/src/agent/position.py
class Position:
    def __init__(self, x: float, y: float, angle: float):
        self.x = x
        self.y = y
        self.angle = angle

    def __str__(self):
        return f"Position(x={self.x}, y={self.y}, angle={self.angle})"
